network/p2p/unicast/plain.go

```diff
@@ -0,0 +1,24 @@
+package unicast
+
+import (
+	libp2pnet "github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/protocol"
+)
+
+// PlainStream is a stream factory that reflects the same input stream without any modification.
+type PlainStream struct {
```

Comment:
perhaps call this `UncompressedStream` to make its name convey more meaning?

Reply:
Going forward, stream manager may serve more than compression. For example, we plan to _experiment_ stream manager for enabling QUIC protocol between limited roles for transferring large data. Considering that, the `PlainStream` may have an overarching general meaning than just an `UncompressedStream`, which is the reason I try not to call it an uncompressed one.